src/overlays/actors/ovl_En_Giant/z_en_giant.c

```diff
@@ -26,42 +31,449 @@ const ActorInit En_Giant_InitVars = {
     (ActorFunc)EnGiant_Draw,
 };
 
-#endif
+extern AnimationHeader D_06002168;
+extern Gfx D_06005A80[];
+extern Gfx D_06006280[];
+extern Gfx D_06007610[];
+extern Gfx D_06006A80[];
+extern FlexSkeletonHeader D_060079B0;
+extern AnimationHeader D_06008394;
+extern AnimationHeader D_060096E4;
+extern AnimationHeader D_0600A1C4;
+extern AnimationHeader D_0600ACA4;
+extern AnimationHeader D_0600B784;
+extern AnimationHeader D_0600C5D4;
+extern AnimationHeader D_0600D040;
+extern AnimationHeader D_0600DE84;
+extern AnimationHeader D_060102A4;
+extern AnimationHeader D_060116E4;
+extern AnimationHeader D_06012A38;
+extern AnimationHeader D_06013004;
+extern AnimationHeader D_06013FE8;
+extern AnimationHeader D_06015334;
+extern AnimationHeader D_06017944;
+
+static AnimationHeader* sAnimationTable[] = {
+    &D_06008394, &D_060096E4, &D_060102A4, &D_060116E4, &D_06012A38, &D_06013004, &D_06013FE8, &D_06015334,
+    &D_06017944, &D_0600A1C4, &D_0600D040, &D_0600DE84, &D_0600ACA4, &D_0600B784, &D_0600C5D4,
+};
+
+static TexturePtr sFaceTextures[] = { D_06005A80, D_06006280, D_06006A80 };
```

Comment:
These can go in the Draw function, most likely. You can add some comments to remind you for when you do the object if that's helpful.

Reply:
They can go in the Draw function so long as they remain `static`; making them non-`static` is not OK. I moved them there and added comments explaining what the three faces are for now.